dyndns53.py

```diff
@@ -169,8 +169,11 @@ def lambda_handler(event, context):
         response = _handler(event, context)
     except Exception as e:
         try:
-            j = {'status':e.status, 'response':e.response, 'additional':e.message}
-        except AttributeError as f:
-            raise type(e)(json.dumps(j)) from e
+            error_info = {'status':e.status, 'response':e.response, 'additional':e.message}
```

Comment:
In Python3 there is no `e.message` attribute triggering an `AttributeError`. Therefore *any* error will return `911` now.

Reply:
Very good observation! Resolved in dba032e by replacing `e.message` with `str(e)`